src/compounds/ItemPage/ItemPage.jsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Button, Row, Col } from 'react-bootstrap'
 import Image from '../../components/Image/Image'
-import TextBox from '../../components/TextBox/TextBox'
 import Title from '../../components/Title/Title'
-import './item-page.css'
 
 const ItemPage = ({ title, titleStyle, description, descriptionStyle, img }) => {
   // sets a default width while still allowing width to be overridden
   img = { width: 400, ...img }
 
   return (
-    <main className='container item-page'>
-      <Title title={title} style={titleStyle} />
-      <div className='item-page-details'>
-        <TextBox text={description} style={descriptionStyle} size='medium' />
```

Comment:
why'd you remove the TextBox? it allowed for additional styling if desired.

Reply:
i removed it because it has an alignment class that was overriding the responsive styling I was trying to do. it wouldn't allow the text to be centered at a lower screen size because it forces you to choose a single alignment. 

<img width="423" alt="image" src="https://user-images.githubusercontent.com/73361970/211829677-e8f9b1f0-e978-4565-93bc-edfc95ac019c.png">

since you want all the text left aligned though instead of using the responsive styles, we can bring it back